test/src/api-action-creator-tests.js

```diff
@@ -36,6 +36,21 @@ describe('APIActionCreators', function () {
         Should.exist(aac.doOtherThing);
     });
 
+    it('should be given a request object after being called', function () {
+        var aac = new APIActionCreator({
+            displayName: 'api45',
+            doThing: {
+                route: '/mirror',
+                method: 'POST',
+                pending: 'TEST_' + Math.random()
```

Comment:
the `random` part?  It's because you are not allowed to have multiple action creators creating the same action (singletons, wooo)... but we still want to be able to copy/paste